python/tests/test_common_estimator.py

```diff
@@ -429,10 +437,11 @@ def assert_model(model: SparkRapidsMLDummyModel) -> None:
         assert dummy2.cuml_params["a"] == 100
         with pytest.raises(
             Exception,
-            match="assert {'a': 9876, 'k': 4, 'x': 40.0} == {'a': 100, 'k': 4, 'x': 40.0}",
+            match="assert {'a': 9876.0, 'k': 4, 'x': 40.0} == {'a': 100, 'k': 4, 'x': 40.0}",
         ):
-            dummy2.fit(df, {dummy2.alpha: 9876})
-        assert dummy2.cuml_params["a"] == 9876
+            dummy2.fit(df, {dummy2.alpha: 9876.0})
```

Comment:
Can the test check that model created has alpha=9876.0?

Reply:
Done.

Reply:
Sorry missed the first time, but why should this raise an exception?

Reply:
the runtime check will enable parameters checking passed from driver side on the executor side.